projects/js-packages/publicize-components/src/social-store/selectors/shares-data.ts

```diff
@@ -149,16 +142,15 @@ export function numberOfSharesRemaining(
 	}
 
 	// Allow partial options to be passed in
-	const { includeScheduled, includeEnabledConnections } = {
+	const { includeScheduled, enabledConnectionsCount } = {
 		includeScheduled: true,
-		includeEnabledConnections: true,
+		enabledConnectionsCount: 0,
 		...options,
 	};
 
 	const sharesUsed = getSharesUsedCount( state );
 	const sharesLimit = getShareLimit( state );
 	const scheduledShares = includeScheduled ? getScheduledSharesCount( state ) : 0;
-	const enabledConnections = includeEnabledConnections ? getEnabledConnections( state ) : [];
 
-	return Math.max( sharesLimit - sharesUsed - scheduledShares - enabledConnections.length, 0 );
+	return Math.max( sharesLimit - sharesUsed - scheduledShares - enabledConnectionsCount, 0 );
```

Comment:
Similarly, I think we won't need this either, because the 'enabled' connections are based on various pieces of information from both the social and editor stores, as well as the validation information. Perhaps we should move that into the store, so it's associated with the connection.